Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/ChromeBrowserContentLocator.cs

```diff
@@ -125,17 +106,18 @@ private Rectangle ResolveBoundsFromWindow (AutomationElement window)
           (int)Math.Round(rawBounds.Height));
     }
 
-    private AutomationElement GetContentElement (AutomationElement window)
+    private AutomationElement? GetContentElement (AutomationElement window)
     {
-      var automationElement = window.FindAll(
-              TreeScope.Subtree,
-              // Matching FrameworkIdProperty == "Chrome" or ControlTypeProperty == "Document" does not work. The value of FrameworkId is instead "Win32".
-              // Since there does not seem to be an issue with detecting the correct window using a simple comparison, no extra conditions are added at this time.
-              new PropertyCondition(AutomationElement.ClassNameProperty, "Chrome_RenderWidgetHostHWND")
+      return window.FindFirst(
+          TreeScope.Children,
+          new AndCondition(
+              new PropertyCondition(AutomationElement.ControlTypeProperty, ControlType.Document),
+              new PropertyCondition(AutomationElement.ClassNameProperty, "Chrome_RenderWidgetHostHWND"),
+              new OrCondition(
+                  new PropertyCondition(AutomationElement.FrameworkIdProperty, "Chrome"),
+                  new PropertyCondition(AutomationElement.FrameworkIdProperty, "Win32"))
```

Comment:
@MichaelKetting Why is this OR condition necessary?

Reply:
@iam-mholle Local vs BuildAgent. shyould add a comment.

Reply:
I can't really comprehend why this would be different on the 2 machines, I thought this is implementation specific for a specific app (-> stays the same for the chrome version used, independent of the machine it is run on). But I guess that's just the way it is then, a comment would really be nice tho, thanks 🙂 .

Reply:
@iam-mholle  Need to amend my original reasoning. The problem is not between build agnet and local system but between UI automation diagnostic tool and actual value read during execution. I added a comment but am keeping both values in the hopes that this will make the code more robust in case the mismatch is actually a bug / caching issue somewhere.